docs/UserGuide.md

```diff
@@ -1,97 +1,299 @@
 
-AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized for use via a Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, AB3 can get your contact management tasks done faster than traditional GUI apps.
-
-* Table of Contents
-{:toc}
-
---------------------------------------------------------------------------------------------------------------------
-
-## Quick start
+## Setup
 
 1. Ensure you have Java `11` or above installed in your Computer.
 
-1. Download the latest `addressbook.jar` from [here](https://github.com/se-edu/addressbook-level3/releases).
+2. Download the latest `spendnsplit.jar` from [here](https://github.com). (Coming soon)
 
-1. Copy the file to the folder you want to use as the _home folder_ for your AddressBook.
+3. Copy the file to the folder you want to use as the _home folder_ for your Spend N Split.
 
-1. Open a command terminal, `cd` into the folder you put the jar file in, and use the `java -jar addressbook.jar` command to run the application.<br>
+4. Open a command terminal, `cd` into the folder you put the jar file in, and use the `java -jar spendnsplit.jar` command to run the application.<br>
    A GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.<br>
    ![Ui](images/Ui.png)
 
-1. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
+5. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
    Some example commands you can try:
 
-   * `list` : Lists all contacts.
+    * `list` : Lists all expenses.
 
-   * `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01` : Adds a contact named `John Doe` to the Address Book.
+    * `clear` : Deletes all content.
 
-   * `delete 3` : Deletes the 3rd contact shown in the current list.
+    * `exit` : Exits the app.
 
-   * `clear` : Deletes all contacts.
+6. Refer to the [Features](#features) below for details of each command.
 
-   * `exit` : Exits the app.
+## Features
 
-1. Refer to the [Features](#features) below for details of each command.
+### __v1.2__
 
---------------------------------------------------------------------------------------------------------------------
+### Adding an expense: `addExpense`
+Adds an expense.
+Format: `addExpense n/NAME c/COST d/DETAILS`
 
-## Features
+Examples:
+* `addExpense n/John Doe c/25 d/Sourdough bread`
+* `addExpense n/Sir Bobby c/1759 d/Iphone 20`
+
+Sample Execution:
+
+```bash
+$ add n/John Doe c/25 d/Sourdough bread 
+
+Added an expense for John Doe. Sourdough bread, $25.00.
+
+$ add n/Ryan d/Sourdough bread 
+
+Error. Expense cost was not provided with a c/ flag.
+```
+![addExpense success](https://hackmd.io/_uploads/Skyr0m2J6.jpg)
```

Comment:
I think you have to download these images, put them in the somewhere in the images folder in our project, and link the path

Reply:
Ok will do so